src/main/java/com/aws/greengrass/shadowmanager/sync/SyncHandler.java

```diff
@@ -89,6 +89,17 @@ public class SyncHandler {
      * Request queue.
      */
     private final RequestBlockingQueue syncQueue;
+    // retry wrapper so that requests can be mocked
```

Comment:
I would remove this reference that it is just for tests, not super important to reference as a comment but it is ok to leave either way